### ISSUE TITLE
Flatpack segfault at startup

### ISSUE THREAD

**Issue by bochecha** [jliljebl/flowblade#350]
I've tried making a Flatpak build of Flowblade, as @sramkrishna requested in #297 .

I've got something that builds, unfortunately as soon as it starts, the window briefly appears and then the app segfaults in MLT.

Here's the backtrace:

```
#0  mlt_service_connect_producer (self=0x0, producer=0x1a81a90, index=0) at mlt_service.c:202
#1  0x00007f3f19ddd5d8 in _wrap_Consumer_connect (args=<optimized out>) at mlt_wrap.cxx:21542
#2  0x00007f3f1b895cef in PyEval_EvalFrameEx () from /lib/libpython2.7.so.1.0
#3  0x00007f3f1b89517e in PyEval_EvalFrameEx () from /lib/libpython2.7.so.1.0
#4  0x00007f3f1b89517e in PyEval_EvalFrameEx () from /lib/libpython2.7.so.1.0
#5  0x00007f3f1b89517e in PyEval_EvalFrameEx () from /lib/libpython2.7.so.1.0
#6  0x00007f3f1b89517e in PyEval_EvalFrameEx () from /lib/libpython2.7.so.1.0
#7  0x00007f3f1b896b1c in PyEval_EvalCodeEx () from /lib/libpython2.7.so.1.0
#8  0x00007f3f1b896c69 in PyEval_EvalCode () from /lib/libpython2.7.so.1.0
#9  0x00007f3f1b8b035f in ?? () from /lib/libpython2.7.so.1.0
#10 0x00007f3f1b8b1462 in PyRun_FileExFlags () from /lib/libpython2.7.so.1.0
#11 0x00007f3f1b8b2567 in PyRun_SimpleFileExFlags () from /lib/libpython2.7.so.1.0
#12 0x00007f3f1b8c1a6e in Py_Main () from /lib/libpython2.7.so.1.0
#13 0x00007f3f1b42c690 in __libc_start_main (main=0x4006d0 <main>, argc=2, argv=0x7ffecf7863a8, init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7ffecf786398)
    at libc-start.c:289
#14 0x0000000000400709 in _start ()
```

I'm not sure how to go on debugging this. :-/

Attached is a zip file containing the necessary stuff to build a flatpaked Flowblade. Just extract it, `cd` into the folder, then run:

```
$ flatpak-builder --ccache --force-clean --keep-build-dirs appdir com.jliljebl.Flowblade.json
```

Once the build finishes, run the app from the build directory:

```
$ flatpak-builder --run appdir/ com.jliljebl.Flowblade.json bash
bash-4.3$ gdb python
(gdb) run /app/bin/flowblade
```

[flowblade-flatpak.zip](https://github.com/jliljebl/flowblade/files/804208/flowblade-flatpak.zip)

**Comment by dvdlvr** [comment]
When I launch it like this I get these errors before the crash. This might be related to the cause, but I don't know how to fix the second one. Obviously these errors do not occur when launching Flowblade normally.
.....
ERROR 1:
Starting program: /usr/bin/python /app/bin/flowblade
warning: Error disabling address space randomization: Operation not permitted
SOLUTION: Enter the following command after issuing gdb python:
set disable-randomization off
This doesn't stop the crash though.
.....
ERROR 2:
** (flowblade:6): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-vJgg7y4BH2: Connection refused

**Comment by bochecha** [comment]
> ** (flowblade:6): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-vJgg7y4BH2: Connection refused

This is just a warning and can be safely ignored. It currently happens on every app in Flatpak, because the accessibility story is still a work in progress.

**Comment by jliljebl** [comment]
I don't know why MLT doesn't work in your Flatpack, so it is difficult to offer any assistance, this obviously does not happen in normal installations of MLT.

**Comment by jliljebl** [comment]
Merged with #453, closing.
